src/main/java/org/frankframework/frankdoc/model/Feature.java

```diff
@@ -15,25 +15,28 @@
 */
 package org.frankframework.frankdoc.model;
 
+import org.frankframework.frankdoc.Constants;
 import org.frankframework.frankdoc.wrapper.FrankAnnotation;
 import org.frankframework.frankdoc.wrapper.FrankClass;
 import org.frankframework.frankdoc.wrapper.FrankDocException;
 import org.frankframework.frankdoc.wrapper.FrankEnumConstant;
 import org.frankframework.frankdoc.wrapper.FrankMethod;
 
-final class Feature {
-	static final Feature MANDATORY = new Feature("nl.nn.adapterframework.doc.Mandatory", "@ff.mandatory");
-	static final Feature OPTIONAL = new Feature("nl.nn.adapterframework.doc.Optional", "@ff.optional");
-	static final Feature DEFAULT = new Feature("nl.nn.adapterframework.doc.Default", "@ff.default");
-	static final Feature DEPRECATED = new Feature("java.lang.Deprecated", "@deprecated");
-	static final Feature PROTECTED = new Feature("nl.nn.adapterframework.doc.Protected", "@ff.protected");
+enum Feature {
+	MANDATORY("nl.nn.adapterframework.doc.Mandatory", Constants.JAVA_DOC_TAG_MANDATORY, new AnnotationValueGetter(Constants.IGNORE_COMPATIBILITY_MODE)),
+	OPTIONAL("nl.nn.adapterframework.doc.Optional", "@ff.optional", new AnnotationValueGetter()),
+	DEFAULT("nl.nn.adapterframework.doc.Default", "@ff.default", new AnnotationValueGetter()),
+	DEPRECATED("java.lang.Deprecated", "@deprecated", new AnnotationValueGetter()),
+	PROTECTED("nl.nn.adapterframework.doc.Protected", "@ff.protected", new AnnotationValueGetter());
```

Comment:
Hier is het overzichtelijker als je 'fieldName' als derde argument gebruikt.

Reply:
Gedaan.